.travis.yml

```diff
@@ -20,7 +20,6 @@ before_install:
   - sudo apt-get install -qq pandoc
 install:
   - pip install -U pip wheel setuptools
-  - pip install -U --use-wheel pytest pytest-cov
```

Comment:
I think you should not delete this. Just change it to `pip install -U pytest pytest-cov` please.